src/main/resources/fester_messages.xml

```diff
@@ -189,4 +189,5 @@
   <entry key="MFS-176">Found item '{}' with format '{}' in CSV '{}', but the access url has not been processed for Wowza</entry>
   <entry key="MFS-177">Test server was responsive: {}</entry>
   <entry key="MFS-178">Looking for test application's server port: {}</entry>
+  <entry key="MFS-179">A/V server URL substitution only supports up to one replacement: {}</entry>
```

Comment:
This makes me curious as to why `WatcherVerticle.addIdPath` in av-pairtree may return a URL template string with two substitution patterns.

Reply:
I started with supplying the `mp4:` part too as a part of the substitutions. I decided that wasn't really necessary for Fester, but didn't see a reason to remove that additional functionality from av-pairtree. Someone might find being able to do substitutions on either side of the ID/Pairtree path useful, depending on their image server.

Reply:
:+1: